aiokafka/consumer/group_coordinator.py

```diff
@@ -401,8 +407,18 @@ async def _perform_assignment(
             'Invalid assignment protocol: %s' % assignment_strategy
         member_metadata = {}
         all_subscribed_topics = set()
-        for member_id, metadata_bytes in members:
+        group_instance_id_mapping = {}
+        # for member_id, group_instance_id, metadata_bytes in members:
+        for member in members:
+            if len(member) == 2:
```

Comment:
This won't work well when we add typing. Could we switch by `isinstance()` instead?

Reply:
I don't think so, since `members` is a member of the `JoinGroupResponse` class [as shown here](https://github.com/joshuaherrera/aiokafka/blob/kip-345/aiokafka/protocol/group.py#L50-L55) and it is defined as an Array. Any version under v5 will be an array of 2, v5 and up currently are an array of 3.

I can add a comment for documentation purposes, so that future users know why we check for a length of two or three, unless you have another suggestion?

Reply:
Another thing I just thought of was modifying the `_perform_assignment` [function signature ](https://github.com/joshuaherrera/aiokafka/blob/kip-345/aiokafka/coordinator/base.py#L189)to just accept the `JoinGroupResponse` as it's only parameter, so we can check the class in this function. So instead the function signature would be 
```python
def _perform_assignment(self, response: JoinGroupResponse):
```    
and we could then use `isinstance` to check the version of the `response` that is being used.

 Another option could be adding the [`API_VERSION`](https://github.com/joshuaherrera/aiokafka/blob/kip-345/aiokafka/protocol/group.py#L41) as a new parameter to the function, so we could just check against that here. So the function signature would become 
 ```python
 def _perform_assignment(self, leader_id, protocol, members, api_version):
 ```

Reply:
I'll work on the first solution, updating the private function's signature to accept a `Response` object, since each `JoinGroupResponse` version implements the `Response` [abstract class](https://github.com/joshuaherrera/aiokafka/blob/kip-345/aiokafka/protocol/api.py#L95).